src/ratelimiter.go

```
@@ -181,36 +173,43 @@ func (i *IPRateLimiter) GetLimiter(ip string) (*rate.Limiter, bool) {
 	
 	// 检查是否在白名单中
 	if isIPInCIDRList(cleanIP, i.whitelist) {
-		return rate.NewLimiter(rate.Inf, i.b), true // 白名单中的IP不受限制
+		return rate.NewLimiter(rate.Inf, i.b), true
 	}
 	
-	// 使用纯IP作为缓存键
+	now := time.Now()
+	
 	i.mu.RLock()
 	entry, exists := i.ips[cleanIP]
```

golangci-lint (1.64.8)

SA4006: this value of `entry` is never used

(staticcheck)